tests/integration/device_test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+include './src/device_info.sh'
+
+function oneTimeSetUp()
+{
+  setup_container_environment
+}
+
+# test if `kw device --local in the container matches the host
+function device_info_test_helper()
+{
+  local distro="${1}"
+  local container
+  local buffer
+  local filter
+  local filter_regex
+  local output
+  local expected_output
+
+  # buffer with all information
+  container="kw-${distro}"
+  buffer=$(container_exec "$container" kw device --local)
+
+  # some fields must be ignored because they surely won't match
+  filter=(
+    'Root filesystem' 'Size' 'Mounted on'                        # storage fields
```

Comment:
I guess we can get this information from the container at run time instead of ignoring it.